presets/4.3/tune/UAV_tech_5in.txt

```diff
@@ -0,0 +1,48 @@
+#$ TITLE: UAV Tech - 5in (~630g AUW)
+#$ FIRMWARE_VERSION: 4.3
+#$ CATEGORY: TUNE
+#$ STATUS: OFFICIAL
```

Comment:
all-new presets should first go to as "EXPERIMENTAL". And later on with some feedback can become community and then official